tox.ini

```diff
@@ -34,18 +45,23 @@ description =
     run tests
     alldeps: with all optional dependencies
     devdeps: with the latest developer version of key dependencies
-    oldestdeps: with the oldest supported version of key dependencies
+    devdeps: with the latest developer version of key dependencies
```

Comment:
This is a mistake? You are duplicating `devdeps`?

Reply:
Oops